newIDE/app/src/MainFrame/Editors/StartPage.js

```diff
@@ -81,4 +94,4 @@ export default class StartPage extends BaseEditor {
       </Column>
     );
   }
-}
+}
```

Comment:
It's a good practice to always have a new line at the end of a file: See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline 😉 
You can configure your text editor to automatically enforce this.